### PR TITLE
[java] Ensure calling close() and quit() done cause BiDi websocket errors

### LINKED ISSUE #13316: [🐛 Bug]: BiDi Connection is established on driver.close / driver.quit
### What happened?

Start a BiDi enabled session, but do not call `driver.getBiDi()`, this will let the driver initialize the connection on `driver.close()` and `driver.quit()`. 

Both calls will run into a `TimeoutException` after ~ 30s, this makes the shutdown of such a driver very slow and exceptions need to be handled.

### How can we reproduce the issue?

```shell
var options = new ChromeOptions();        
options.setCapability("webSocketUrl", true);
Augmenter augmenter = new Augmenter();
WebDriver driver = augmenter.augment(new RemoteWebDriver(uri, options));

try {
    DevTools devTools = ((HasDevTools) driver).getDevTools();
    devTools.createSession();

    driver.close();
} catch (Exception e) {
    e.printStackTrace();
} finally {
    try {
        driver.quit();
    } catch (Exception ex) {
        ex.printStackTrace();
    }
}
```

### Relevant log output

```shell
org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException
Build info: version: '4.16.0', revision: '15400459dd*'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.1'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.bidi.Connection.sendAndWait(Connection.java:172)
	at org.openqa.selenium.bidi.BiDi.send(BiDi.java:54)
	at org.openqa.selenium.bidi.BiDi.<init>(BiDi.java:37)
	at org.openqa.selenium.bidi.BiDiProvider.lambda$getImplementation$2(BiDiProvider.java:55)
	at org.openqa.selenium.remote.RemoteWebDriver$ByteBuddy$3k6C8irl.maybeGetBiDi(Unknown Source)
	at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:435)
	at some.issues.Main.lambda$main$0(Main.java:39)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
	at org.openqa.selenium.bidi.Connection.sendAndWait(Connection.java:161)
	... 11 more
```

### Operating System

win 10 x64

### Selenium version

4.16.1

### What are the browser(s) and version(s) where you see this issue?

chrome 120

### What are the browser driver(s) and version(s) where you see this issue?

chromedriver 120

### Are you using Selenium Grid?

4.16.1

### PR DESCRIPTION
**Pull request by pujagani**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Fixes #13316
If the BiDi capability is passed, then WebDriver that implements HasBiDi interface, which will create the websocket connection for BiDi. So, on WebDriver quit(), those resources do need the clean up.

### Motivation and Context

If WebDriver close() is called, it closes the session if it is the last browsing context. It also closes the WebSocket.
If WebDriver quit() is called after, it also tries to close an already closed websocket. The changes fix that.
This measure is needed until "session.end" from BiDi is implemented by the browsers, which can then be used to close the BiDi session. Similar, to what we do we CDP and dettach target.

Additionally, removed the status check while creating BiDi session, the status seems important if we create BiDi only session, which is not in scope for us currently.
  
### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: java/src/org/openqa/selenium/manager/SeleniumManager.java [hunk 1]
```diff
@@ -258,8 +258,8 @@ public Result getDriverPath(Capabilities options, boolean offline) {
     List<String> arguments = new ArrayList<>();
     arguments.add("--browser");
     arguments.add(options.getBrowserName());
-    arguments.add("--language-binding");
-    arguments.add("java");
```
Comment (diemol) [inline java/src/org/openqa/selenium/manager/SeleniumManager.java]: This should work well if you update the branch.

File: java/src/org/openqa/selenium/manager/SeleniumManager.java [hunk 2]
```diff
@@ -258,8 +258,8 @@ public Result getDriverPath(Capabilities options, boolean offline) {
...
+    //    arguments.add("--language-binding");
```
Comment (krmahadevan) [inline java/src/org/openqa/selenium/manager/SeleniumManager.java]: Is this intentionally commented out?

**Review by diemol — CHANGES_REQUESTED** [review]
Looks good, but seems some commented code was left in the changeset :)

File: java/src/org/openqa/selenium/manager/SeleniumManager.java (hunk 2 above)
Comment (pujagani) [reply java/src/org/openqa/selenium/manager/SeleniumManager.java]: Yes, Selenium Manager work was in progress and locally I commented it out to ensure I can run tests. I think it is fixed now.

File: java/src/org/openqa/selenium/manager/SeleniumManager.java (hunk 1 above)
Comment (pujagani) [reply java/src/org/openqa/selenium/manager/SeleniumManager.java]: Oops. Yes, thank you!

**Pull request merged**